src/app/services/baw-api/baw-api.service.ts

```diff
@@ -691,6 +807,17 @@ export class BawApiService<
       },
     };
   }
+
+  private addUnscopedParams(
+    data: FormData,
+    params: BawServiceOptions["params"]
+  ): FormData {
+    for (const [key, value] of Object.entries(params)) {
+      data.append(key, value);
+    }
```

Comment:
seems we just special cased adding arrays to formdata, but we haven't done it here. is that an issue?

Reply:
Really good catch (thank you).

TypeScript should have caught this so I think an underlying issue here is that the `params` property on the options is a `Record<any, any>` so I didn't get any TypeScript errors to warn me about this.

![image](https://github.com/user-attachments/assets/9596dc3e-d1a2-44f0-a9ca-32bc7fd469d1)